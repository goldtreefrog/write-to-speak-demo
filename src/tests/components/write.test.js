import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import Write from "./../../components/write.js";

it("renders without crashing and matches snapshot", () => {
  const wrapper = shallow(<Write />);
  expect(wrapper).toMatchSnapshot();
});
