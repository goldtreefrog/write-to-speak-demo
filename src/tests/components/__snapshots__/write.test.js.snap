// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing and matches snapshot 1`] = `
<Redirect
  push={false}
  to="/login"
/>
`;
